pkg/dbdump/dbdump.go

```diff
@@ -0,0 +1,280 @@
+package dbdump
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/noobaa/noobaa-operator/v5/pkg/options"
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+	"github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+// Cmd returns a CLI command
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "db-dump",
+		Short: "Collect db dump",
+		Run:   RunDump,
+		Args:  cobra.NoArgs,
+	}
+	cmd.Flags().String("dir", "", "collect db dump file into destination directory")
+	return cmd
+}
+
+// Collector configuration for diagnostics
+type Collector struct {
+	folderName			string // Local folder to store the raw db dump
+	dbDumpFolderPath	string // Local path of the folder containing the raw dump file
+	remoteFolderPath	string // Remote folder (at pod) to store the raw db dump
+	remoteTarPath		string // Remote path (at pod) of tar'd db dump
+	kubeconfig 			string
+	log					*logrus.Entry
```

Comment:
Indentation

Reply:
To me this seems like a Git UI issue, as the code looks well formatted in my workspace

Reply:
Yes, also looks ok for me in vs code.